runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -470,6 +471,13 @@ ClassFileOracle::walkAttributes()
 						markConstantUTF8AsReferenced(entry->innerNameIndex);
 						_simpleNameIndex = entry->innerNameIndex;
 					}
+				/* Count all entries in the InnerClass attribute (except the inner class itself) so as
```

Comment:
This comment should be after the `} else }`

Reply:
Fixed.